Makefile

```diff
@@ -0,0 +1,57 @@
+# Run `make help` to display help
+
+power:  ## Set up power.jul.run
+	kubectl apply -f power
```

Comment:
should you also add kubectl to hermit in this commit?

Reply:
ok as per discussion - leaving it in first commit.